werkzeug/exceptions.py

```diff
@@ -672,7 +672,27 @@ def __call__(self, code, *args, **kwargs):
             raise LookupError('no exception for %r' % code)
         raise self.mapping[code](*args, **kwargs)
 
-abort = Aborter()
+
+def abort(status, *args, **kwargs):
```

Comment:
so the new contract of abort will always be status?

we use kwargs and we have issue updating since we were doing:
`abort(code=400, description="Invalid destination name")`

with this commit it raise 
`TypeError: abort() missing 1 required positional argument: 'status'`

I don't know if this was the expected behavior if we we should also change this line
https://github.com/pallets/werkzeug/pull/1003/files#diff-43a63db82587e91732eda181306d76c7R668